components/script/dom/htmlelement.rs

```diff
@@ -1186,6 +1198,14 @@ impl VirtualMethods for HTMLElement {
         }
     }
 
+    fn post_connection_steps(&self) {
+        if let Some(s) = self.super_type() {
+            s.post_connection_steps();
+        }
+
+        self.upcast::<Element>().update_nonce_post_connection();
```

Comment:
Would it make more sense to call this from Element::post_connection_steps?

Reply:
In terms of code sharing, yes. The reason I duplicated these two is that the spec is explicit in saying it should be put on the `HtmlOrSvgMixin`. I don't think it makes a functional difference to do it in `Element`, so how do you interpret the spec here?